crates/client/rpc-core/src/utils.rs

```diff
@@ -61,3 +64,23 @@ pub(crate) fn _to_deprecated_entrypoint_by_type(
     });
     DeprecatedEntryPointsByType { constructor, external, l_1_handler }
 }
+
+pub fn to_invoke_tx(tx: BroadcastedInvokeTransaction) -> Result<InvokeTransaction> {
```

Comment:
can we switch this to a `TryInto` trait impl ?

Reply:
I can't add a `TryInto` outside the crate of the type delaraction (i.e. rpc-core)

I tried to add a `TryInto` in `primitives/transaction` but the issue is `starknet-providers` doesn't expose a no-std feature